### PR TITLE
164434238: build out Post Rest API Endpoint

### PR DESCRIPTION
**Pull request by Chiazokam**
### What does this PR do?
builds out Post Rest API Endpoint

### CODE REVIEW

**Comment by amaechi-chuks** [comment]
> ### What does this PR do?
> Creates the create note endpoint

I think this title should be 'build out Post Rest API Endpoint'

**Comment by sojida** [comment]
Great Job. I think this should be merged

File: server/src/controllers/notes.js [hunk 1]
```diff
@@ -12,6 +12,18 @@ class Notes {
     });
   }
 
+  static async createNote(req, res) {
+    const Note = await notes.create({
+      title: req.body.title,
```
Comment (vic3king) [inline server/src/controllers/notes.js]: This looks Nice and works, But you can consider using destructuring for the req.body to simplify some more.

Comment (Chiazokam) [reply server/src/controllers/notes.js]: Alright..thanks

**Pull request merged**